App.js

```diff
@@ -1,61 +1,83 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
-import { Provider } from 'react-redux';
+import { Button } from 'react-native';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import { logoutUser } from './components/Auth/Services/client';
+import {
+  authenticateUser,
+  setUserAuthenticationToken,
+} from './redux/authSlice';
 import { store } from './redux/store';
 
+import HomeScreen from './screens/HomeScreen';
 import LandingScreen from './screens/LandingScreen';
 import LoginInScreen from './screens/LoginScreen';
 import SignUpScreen from './screens/SignUpScreen';
 
 const Stack = createNativeStackNavigator();
 
-//* once auth is set up, need to implement conditional statements where it checks if user has a saved profile on the phone
-//* if so, then app should render out the home screen as default
+//* if user is authenticated then authenticated stack should be used, otherwise use auth stack
+//* if token is present then user is authenticated -> authenticated stack
```

Comment:
nit: you should either do:
```
// if user is authenticated then authenticated stack should be used, otherwise use auth stack
// if token is present then user is authenticated -> authenticated stack
```

or use a multiline comment
```
/* if user is authenticated then authenticated stack should be used, otherwise use auth stack
if token is present then user is authenticated -> authenticated stack */

```